app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -121,6 +126,7 @@ class ConversationViewModel @Inject constructor(
 
     init {
         observeConversationDetailsAndMessages()
+        listenIfSelfIsConversationMember()
```

Comment:
Should it be `observe`, instead of `listen` ? to have the same pattern 🤔

Reply:
Will change it in another PR